### PR TITLE
[No QA] Add polyfill for `replaceAll()` to fix Safari iOS 13

### PR DESCRIPTION
**Pull request by marcaaron**
### Fixed Issues (Related to)
$ https://github.com/Expensify/App/pull/8896#issuecomment-1130401493

# Tests

Can be tested in connection with -> https://github.com/Expensify/App/pull/9060

# QA
No QA

### CODE REVIEW

File: lib/str.js [hunk 1]
```diff
@@ -1049,6 +1050,18 @@ const Str = {
...
+
+    /**
+     * Polyfill for String.prototype.replaceAll
+     *
+     * @param {String} text
+     * @param {String|RegExp} searchValue
+     * @param {String|Function} replaceValue
+     * @returns {String}
+     */
+    replaceAll(text, searchValue, replaceValue) {
```
Comment (techievivek) [inline lib/str.js:1062]: Just kind of curious why we have to import a package for this, why can't we use inbuilt [replaceAll](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll)?

Comment (marcaaron) [reply lib/str.js:1062]: Mostly just to backfill support for browsers where it's not available

https://caniuse.com/?search=replaceAll

Comment (marcaaron) [reply lib/str.js:1062]: We could implement a polyfill in every repository, but I think it is just easier in this case to add it to our "string utility library" which several other repos are using.

Comment (techievivek) [reply lib/str.js:1062]: Thanks for clearing this out I can see ios Safari `3.2 - 13.3` is not supported. 🙌

**Review by techievivek — APPROVED** [review]
Looks good.

**Pull request merged**